src/components/Image/Image.tsx

```diff
@@ -0,0 +1,68 @@
+import { CSSProperties } from 'react';
+import { makeStyles } from '../../utils/makeStyles';
+import { sx } from '../../utils/sx';
+import { BaseStyleProp } from '../types';
+
+type LinkStyles = 'root' | 'image' | 'caption';
+
+export interface ImageProps extends BaseStyleProp<LinkStyles> {
+  src: string;
+  alt: string;
+  caption?: string;
+  width: CSSProperties['width'];
+  height?: CSSProperties['height'];
+  imageAlign?: 'left' | 'center' | 'right';
+  captionAlign?: 'left' | 'center' | 'right';
+}
+
+const useStyles = makeStyles({
+  root: { border: '0', display: 'block', outline: 'none' },
+  table: { width: 'fit-content', border: '0', verticalAlign: 'top' },
+  image: {},
+  caption: {},
+});
+
+export const Image = ({
+  src,
+  alt,
+  caption,
+  width,
+  height = 'auto',
+  classes,
+  className,
+  imageAlign = 'center',
+  captionAlign = 'center',
+}: ImageProps): JSX.Element => {
+  const styles = useStyles({ classes });
+
+  return caption ? (
+    <table cellPadding="0" cellSpacing="0" role="presentation" style={styles.table}>
+      <tbody>
+        <tr>
```

Comment:
Let’s add classes to other elements also like `classes.row1`

Reply:
Added classes for other elements following the `Section` and `Column` naming convention.